Projects/TerminateContracts/Sources/Service/OctopusImplementation/TerminateContractsServiceOctopus.swift

```diff
@@ -10,7 +10,62 @@ public class TerminateContractsOctopus: TerminateContractsService {
             input: OctopusGraphQL.FlowTerminationStartInput(contractId: contractId),
             context: nil
         )
-        return try await mutation.execute(\.flowTerminationStart.fragments.flowTerminationFragment.currentStep)
+        //        return try await mutation.execute(\.flowTerminationStart.fragments.flowTerminationFragment.currentStep)
```

Comment:
remove comment

Reply:
We need this when we get the implementation from the BE